typing/ikinds/ikinds.mli

```diff
@@ -0,0 +1,62 @@
+val ikind_reset : string -> Types.type_ikind
+
+val normalize : context:Jkind.jkind_context -> Types.jkind_l -> Ikind.Ldd.node
+
+val type_declaration_ikind :
+  context:Jkind.jkind_context -> path:Path.t -> Types.constructor_ikind
```

Comment:
Is there a reason to expose these two functions? They're not used anywhere outside ikinds.ml